### DISCUSSION TITLE
calc vs qalculate?

### DISCUSSION THREAD

**Discussion by ghost** [lcn2/calc#127] [Q&A] [answered]
https://github.com/Qalculate/libqalculate

**Comment by pmetzger** [comment]
@iahunq2 What is your question?

**Reply by ghost** [reply]
It seems qalculate is a more advanced calculator but calc is better for scripting. I want to know how is calc compared to qalculate from experienced calc users.

**Reply by pmetzger** [reply]
I have no opinion.

**Reply by ghost** [reply]
I will wait for @lcn2.

**Comment by pmetzger** [comment]
Looks like the person who asked the question (with a brand new account) deleted their account.

**Comment by lcn2** [comment, marked as answer]
We are not interested in doing the work to study some other application and then write a comparison.  If someone wants to understand the difference between two applications, they can study them both.

It is not reasonable to expect calc developers to know about other calculating applications.  Calc is well documented.  Perhaps some other application is it is not as well.  Unless we are already familiar with the other application, we do not have an ability to comment nor do we want to take the time to study the other application in order to help you do a “vs” comparison.  

Sorry.
